setup.py

```diff
@@ -14,13 +14,26 @@
 # You need to have the setuptools module installed. Try reading the setuptools
 # documentation: http://pypi.python.org/pypi/setuptools
 REQUIRES = ["requests >= 2.0.0", "six", "pytz", "PyJWT >= 1.4.2"]
+REQUIRES_PY2 = ["cryptography >= 1.3.4", "idna >= 2.0.0", "pyOpenSSL >= 0.14"]
+REQUIRES_PY3 = ['pysocks']
 
-if sys.version_info < (2, 6):
-    REQUIRES.append('simplejson')
-if sys.version_info < (3, 0):
-    REQUIRES.extend(["cryptography >= 1.3.4", "idna >= 2.0.0", "pyOpenSSL >= 0.14"])
-if sys.version_info >= (3, 0):
-    REQUIRES.append('pysocks')
+if 'bdist_wheel' not in sys.argv:
```

Comment:
Do you know what versions of `pip` need this block? Maybe we can remove it if those versions of `pip` are sufficiently old?